components/util/Creator.js

```diff
@@ -4,14 +4,15 @@ const Creator = ({ info, size }) => {
     ? `https://unavatar.io/twitter/${avatarKey}`
     : `https://unavatar.io/${avatarKey}`;
   return (
-    <div className={`creator ${size}`}>
+    <div className="tw-flex tw-mt-5px tw-items-center tw-justify-center">
       <img
         src={creatorImage}
+        className="tw-rounded-half tw-mr-5px"
         alt="Profile Photo"
         height={size === 'sm' ? '30' : '80'}
         width={size === 'sm' ? '30' : '80'}
       />
-      <span>
+      <span className={(size==='sm' ? "tw-text-18px":"tw-text-30px")}>
```

Comment:
Creator name is not blue like on the production site

Reply:
It is the default Colour of an Anchor `<a> </a>` tag, usually we use custom color but I'll put down the default manually as per you need.